site/content/en/docs/tasks/run_flux.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Run A Flux Minicluster"
+date: 2022-02-14
+weight: 6
+description: >
+  Run a Kueue scheduled Flux minicluster.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Flux Operator's](https://flux-framework.org/flux-operator/) Mini Clusters.
+
+For more information on Flux Operator visit [Flux Operator](https://flux-framework.org/flux-operator/).
+
+
+## Before you begin
+
+Please check [Run A Job](/docs/tasks/run_jobs) for details on the initial cluster setup and queue configuration.
+
+## 1. MiniCluster definition
+
+Since a Flux MiniCluster uses  a `batch/job` to run the actual workload, no extra components are needed for it to be managed by Kueue, However the following aspects should be taken into account:
+
+### a. Queue selection
+
+The target local queue should be specified in the `spec.jobLabels` section of the minicluster configuration.
+
+```yaml
+  jobLabels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+### b. Configure the resource needs
+
+The resource needs of the workload can be configured in the `spec.container[].resources` sections of the minicluster configuration.
+
+```yaml
+spec:
+  containers:
+    - image: <image>
+      resources:
+        requests:
+          cpu: 4
+          memory: "200Mi"
+```
+
+## 2. Configuration example
+
+```yaml
+apiVersion: flux-framework.org/v1alpha1
+kind: MiniCluster
+metadata:
+  generateName: flux-sample-kueue-
+spec:
+  size: 1
+  containers:
+    - image: ghcr.io/flux-framework/flux-restful-api:latest
+      command: echo hello world 
+      resources:
+        requests:
+          cpu: 4
+          memory: "200Mi"
+  jobLabels:
+    kueue.x-k8s.io/queue-name: user-queue
+```
+
+## 3. Known Limitations
+
+- The certificate generation step of the MiniCluster life cycle runs outside it's main job and it's execution will not be managed by Kueue.
```

Comment:
```suggestion
- The certificate generation step of the MiniCluster life cycle runs outside its main job and its execution will not be managed by Kueue.
```

Assuming these are referring to the attributes that belong to the certificate generation